trapperkeeper/callbacks.py

```diff
@@ -64,7 +64,7 @@ def _send_mail(self, handler, trap, is_duplicate):
         ctxt = dict(trap=trap, dest_host=self.hostname)
         try:
             stats.incr("mail_sent_attempted", 1)
-            send_trap_email(recipients, "trapperkeeper",
+            send_trap_email(recipients, "trapperkeeper@localhost",
```

Comment:
This should probably be turned into a config option. I currently use a mail relay that automatically appends the domain so this would negatively affect my installation as proposed. If you want to change this to be a config option with the default remaining `trapperkeeper` then I'll gladly merge this.